.travis.yml

```diff
@@ -14,3 +14,7 @@ env:
 script: bundle exec ./go ci_build
 notifications:
   webhooks: http://project-monitor.cf.18f.us/projects/hub/status
+
+script: bundle exec rake run_about_yml_check
```

Comment:
I'd think the 18F/about_yml#71 is sufficient without needing to link to a commit, but you can just keep this original commit in the branch history and it'll stay alive.